packages/ros2idl-parser/src/parseRos2idl.ts

```diff
@@ -18,6 +18,14 @@ export function parseRos2idl(messageDefinition: string): MessageDefinition[] {
     for (const field of def.definitions) {
       field.type = normalizeName(field.type);
     }
+    // need to correct the builtin_interfaces/msg/Time nanosec field to nsec so that studio can use it
```

Comment:
"studio can use it" is vague - maybe adopt some language from: https://github.com/foxglove/studio/blob/main/packages/mcap-support/src/parseProtobufSchema.ts#L28-L31

And surface the fact that studio makes some assumptions about the field names for "Time" fields to simplify downstream logic.

Reply:
Updated - also looking at the protobuf example made me realize I was missing duration as well. Thanks for that :D